Source/Templates/GtkSharp.Template.CSharp/content/GtkSharp.Application.CSharp/GtkNamespace.csproj

```diff
@@ -13,7 +16,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="GtkSharp" Version="3.24.24.*" />
+    <PackageReference Condition="'$(UseWorkload)' != 'true'" Include="GtkSharp" Version="3.24.24.*" />
+    <PackageReference Condition="'$(UseWorkload)' == 'true'" Include="GtkSharp" Version="$(GtkSharpVersion)" />
```

Comment:
The new workload in this PR will not provide the references, you will need the NuGets. But now they can be updated from a nightly or just bumped independent of the workload version.